### DISCUSSION TITLE
How can I create row after reading a def file

### DISCUSSION THREAD

**Discussion by titan73** [The-OpenROAD-Project/OpenROAD#5324] [Q&A] [answered]
I have a .def file containing the hard ip shape, pins and also one macro placed.
After reading it with read_def. Everything is fine but I have no rows. How can I create the rows?
Note that after using initialize_floorplan, I don't have any rows either.

**Comment by titan73** [comment]
Ah. Correction. initialize_floorplan creates the rows but there are not visible in the fit view. I can see them when zooming. Is that normal? How can I see them independantly from the zoom factor?
Btw, What the vertical lines are supposed to mean?

**Reply by maliberty** [reply, marked as answer]
The vertical lines show the sites within the rows when zoomed in closely.

Misc->Detailed view will draw more shapes when zoomed out at the expense of slower render times.
